backend/core/email.py

```
@@ -0,0 +1,33 @@
+import os
+import logging
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+load_dotenv()
+
+BREVO_API_KEY = os.getenv("BREVO_API_KEY")
+EMAIL_FROM = os.getenv("EMAIL_FROM")
+
+async def send_email(to: str, subject: str, body: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `send_email`

(ANN201)